# HG changeset patch
# Parent f63659c5a2c060ac43bf681191d1432cbb7c3e54
We need to package toolkit locales explicitely into firefox langpacks even in xulrunner based builds

diff --git a/browser/makefiles.sh b/browser/makefiles.sh
--- a/browser/makefiles.sh
+++ b/browser/makefiles.sh
@@ -77,16 +77,21 @@ browser/devtools/styleinspector/Makefile
 browser/devtools/webconsole/Makefile
 browser/fuel/Makefile
 browser/fuel/public/Makefile
 browser/fuel/src/Makefile
 browser/installer/Makefile
 browser/locales/Makefile
 browser/modules/Makefile
 browser/themes/Makefile
+netwerk/locales/Makefile
+dom/locales/Makefile
+security/manager/locales/Makefile
+extensions/spellcheck/locales/Makefile
+toolkit/locales/Makefile
 $MOZ_BRANDING_DIRECTORY/Makefile
 $MOZ_BRANDING_DIRECTORY/content/Makefile
 $MOZ_BRANDING_DIRECTORY/locales/Makefile
 "
 
 if [ "$MOZ_SAFE_BROWSING" ]; then
   add_makefiles "
     browser/components/safebrowsing/Makefile
